File: python/calculate_mode_spec.py
# Write your own unit tests here!
from calculate_mode import calculate_mode

print(calculate_mode([1,2,3,2,5]) == 2)
print(calculate_mode([22,1,45,987,2,22]) == 22)
print(calculate_mode(["red","box", "cat", "poe", "poe"]) == "poe")
print(calculate_mode(['nice', 'nice', 'red', 'nice', 99]) == "nice")
print(calculate_mode(["w", 4, "w", 4, "w", "w", "w", 4, 4, 4, 4]) == 4)

File: js/calculateModeSpec.js
// Write your own unit tests here!
var modeCheck = require("./calculateMode");

console.log(modeCheck.calculateMode([1,2,3,4,4,4,5]) === 4);
console.log(modeCheck.calculateMode([45,8,23,99,23,3,4,1,34]) === 23);
console.log(modeCheck.calculateMode(["w","a","w","d"]) === "w");
console.log(modeCheck.calculateMode(["box", "fox", "mox", "fox", "fox", "lox"]) === "fox");
console.log(modeCheck.calculateMode(["cat", 5, "red", "cat", 5, 6, "cat"]) === "cat");